src/datachain/progress.py

```diff
@@ -148,6 +26,8 @@ def __init__(self, tqdm_kwargs=None, *args, **kwargs):
         self.files_count = 0
         tqdm_kwargs = tqdm_kwargs or {}
         tqdm_kwargs.setdefault("postfix", {}).setdefault("files", self.files_count)
+        kwargs = kwargs or {}
+        kwargs["tqdm_cls"] = tqdm
```

Comment:
```suggestion
        kwargs.setdefault("tqdm_cls", tqdm)
```

Reply:
it's not the same though - it will raise if kwargs is None, right?

Reply:
kwargs would never be None, as it's a variable keyword arguments. It is always going to be a dict.

Reply:
yep, makes sense ... feel free to do a separate minor PR :)